### ISSUE TITLE
Lazy loading and search without ajax call

### ISSUE THREAD

**Issue by s4sari** [vakata/jstree#1424]
Hi,

I have a javascript variable with json data stored from the server.

I would like to use this variable to do lazy load and search without making the ajax call.

I tried using the function approach but frankly I am not able to make heads and tail out of this approach and how it works actually,blame my stupid brain!!!

Request your help to make this work

Related jsfiddle : http://jsfiddle.net/xxbpmcdf/

In short I want all the normal jsTree functionalities but instead of ajax I just want use a local javascript variable.

Thanks in Advance...

**Comment by vakata** [comment]
Here you go:
http://jsfiddle.net/xxbpmcdf/2/

Please use the discussion group for further how-to questions.

Best regards,
Ivan

**Comment by s4sari** [comment]
Wow,that was fast...Thank U!

As suggested I'll use the discussion group for further discussion on this :)
